olive/passes/onnx/conversion.py

```diff
@@ -195,13 +195,15 @@ def _export_pytorch_model(
 
             # handle dummy inputs for model with past, which has past_key_values
             # match input names in `past_key_values.(hidden_layer_num).(key|value)` pattern(llama2 case)
-            # or `past_key_values.(hidden_layer_num)` pattern (phi2 case)
+            # or `past_key_(hidden_layer_num)`| `past_value_(hidden_layer_num)` pattern (phi2 case)
+            import re
+
             for name, dm_input in dummy_inputs.items():
                 if name == "past_key_values" and isinstance(dm_input, list):
```

Comment:
does the name need to be updated?

Reply:
This name is the argument from huggingface model class. I can add more comments to tell why the name should be `past_key_values`

E.g.:
1. phi2
![image](https://github.com/microsoft/Olive/assets/13343117/5b839cc1-63de-4c13-ab9e-a78c5dde9941)

2. mistral.
![image](https://github.com/microsoft/Olive/assets/13343117/71cd374f-1b2c-4237-8abf-4a07f19fef3c)

Reply:
just a little confusion what's the new name in Windows?

Reply:
Currently it is same in linux and windows. Previously, the differences was led by:
In windows, we cannot use dynamo which fallbacks to get_io_config function where we already set the input name. And the input name is different with the one in dynamo. 

Update the name in user_script.py can fix.
![image](https://github.com/microsoft/Olive/assets/13343117/edbbccd5-9d90-425d-9e57-52586afbcca1)